diktat-rules/src/test/kotlin/com/saveourtool/diktat/ruleset/chapter3/ClassLikeStructuresOrderRuleWarnTest.kt

```diff
@@ -196,6 +196,37 @@ class ClassLikeStructuresOrderRuleWarnTest : LintTestBase(::ClassLikeStructuresO
         )
     }
 
+    @Test
+    @Tag(WarningNames.BLANK_LINE_BETWEEN_PROPERTIES)
+    fun `should not trigger on EOL comment on the same line with property`() {
+        lintMethod(
+            """
+                    |class ActiveBinsMetric(meterRegistry: MeterRegistry, private val binRepository: BinRepository) {
+                    |    companion object {
+                    |        private const val EGG_4_5_BUCKET_LABEL = "4-5"
```

Comment:
I'm not sure that test requires so many lines. can be simplified to 1-2 fields

Reply:
Done